docs/api.md

```diff
@@ -4552,4 +4814,50 @@ These are in alphabetic order, with the exception that the `Extended` versions o
 ## Property Detailed (object)
 + name: `first_name` (string, required) - The name of the Property.
 + publishedAt: `2018-01-21T00:04:11.153Z` (string, required) - Publishing timestamp of the form that defined this property for the first time.
-+ forms: (array[Form KeyValue]) - List of forms that create the property
\ No newline at end of file
++ forms: (array[Form KeyValue]) - List of forms that create the property
+
+## EntitySummaryFields (object)
++ uuid: `uuid:85cb9aff-005e-4edd-9739-dc9c1a829c44` (string, required) - The `uuid` of the Entity that uniquely identifies the Entity.
++ datasetName: `people` (string, required) - Name of the dataset
++ createdAt: `2018-01-19T23:58:03.395Z` (string, required) - ISO date format. The time that the server received the Entity.
++ updatedAt: `2018-03-21T12:45:02.312Z` (string, optional) - Timestamp of the last update in ISO date format. `null` when there is only one version of the Entity.
++ deletedAt: `2018-03-21T12:45:02.312Z` (string, optional) - Timestamp of the deletion in ISO date format. `null` if the Entity is not deleted.
+
+## EntitySourceType (enum)
++ SUBMISSION (string) - Entity was created through a Form Submission.
++ API (string) - Entity was created using the API.
++ FILE_UPLOAD (string) - Entity was created through uploading a file.
+
+## EntitySource (object)
++ type: `API` (EntitySourceType, required) - From what type of source this Entity was created.
++ id: `uuid:85cb9aff-005e-4edd-9739-dc9c1a829c44` (string, required) - Submission ID if the type is Submission otherwise value of `X-Client-Id` provided at the time of Entity creation.
+
+## EntityVersionFields (object)
++ versionNumber: `1` (number, required) - Numeric number to identify the version (auto incrementing)
```

Comment:
Alternative: sha1 of the data

Reply:
I like the idea of an ID like `versionNumber` rather than a SHA! Conceivably someone could change the entity data, then change it back again, so a SHA of the data wouldn't be a unique identifier.

Reply:
I like the simplicity of versionNumber, but we may need it to conflict resolution of offline entities. SHA could be an additional field, I haven't thought about it thoroughly though.

Reply:
I feel like some unique ID is needed for each entity version for the diff response. I don't have any preferences though on what that unique ID looks like, whether it's auto-incrementing or a UUID that Central generates or something else.

Reply:
You could also return the diffs as an array rather than an object indexed by the version number/ID, and Frontend could figure it out. That'd be different from submission diffs, but it wouldn't require a version ID.